tutorials/abap-environment-create-abapdoc/abap-environment-create-abapdoc.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Create ABAPDoc Comments in Your Class in SAP Cloud Platform, ABAP Environment
+description: Learn how to maintain ABAPDoc documentation for your class
```

Comment:
This description actually says less than the title!! Make title:

Create ABAPDoc Comments in Your Class

For the description, mention SAP Cloud Platform, ABAP Environment and make it more descriptive, maybe:

Learn how to maintain ABAPDoc documentation for your class in SAP Cloud Platform, ABAP Environment so your comments appear in the Outline view.

I assumed that adding comments and having them appear in the view were the coolest things the user would do, and therefore added them to the description.